cfpb/django-college-costs-comparison
# Remove haystack dependency

The comparison tool is no longer in use, but the repo needs to stay active for now to serve three subpages that will be moved into Wagtail later. We no longer need to index content or require haystack, so the search index and requirement can be removed.

Also, we don't need any of the api view code, so the urls, views and tests are also dropped.

Once this is deployed, we can drop/archive the comparisontool tables from our database and lighten our prod dump.

## Testing
- In a cfgov environment, call `pip uninstall comparisontool` and then Install this branch of the repo with
```bash
pip install -e [PATH_TO_CLONE]
```
- Run this management command:
```bash
./cfgov/manage.py haystack_info
```
It should report only 4 indexes:
```bash
  - Model: AnswerPage by Index: 
  - Model: School by Index: 
  - Model: SectionParagraph by Index: 
  - Model: ActivityPage by Index: 
```

- confirm that these comparisontool pages still render:
    - http://localhost:8000/paying-for-college/choose-a-student-loan/#o1
    - http://localhost:8000/paying-for-college/manage-your-college-money/#o1
    - http://localhost:8000/paying-for-college/repay-student-debt/
